src/other/utils.jl

```diff
@@ -75,3 +75,7 @@ function funname(f)
     n = nameof(f)
     String(n)[1] == '#' ? :function : n
 end
+
+if VERSION >= v"1.6.0-DEV.85"
+    funname(c::Base.ComposedFunction) = Symbol(funname(c.f), :_, funname(c.g))
+end
```

Comment:
I'm not sure if you like a tricky code, but I think you can do this

```suggestion
if isdefined(Base, :ComposedFunction)
    using Base: ComposedFunction
else
    const ComposedFunction = let h = identity ∘ convert
        @assert h.f === identity
        @assert h.g === convert
        getfield(parentmodule(typeof(h)), nameof(typeof(h)))
    end
    @assert identity ∘ convert isa ComposedFunction
end

funname(c::ComposedFunction) = Symbol(funname(c.f), :_, funname(c.g))
```

to make `funname(sum ∘ skipmissing)` to work in all Julia versions back to 1.0.

Reply:
@nalimilan - what do you think? The benefit is that we would then consistently work on every version of Julia we support. The trick uses an undocumented functionality but it seems it is valid :).

Reply:
Why not. The behavior is unlikely to change in existing releases.